src/postman/__tests__/service.test.ts

```diff
@@ -0,0 +1,24 @@
+import { CollectionDefinition } from 'postman-collection';
+import { transformPostmanCollectionService } from '../service';
+
+describe('postman collection service', () => {
+  const postmanCollection = require('./fixtures/petstore.json') as CollectionDefinition;
```

Comment:
```
Type '{ "id": string; "type": string; "value": string; }[]' has no properties in common with type 'VariableDefinition'.
```

- looks like the types are not necessarily reflecting the actual Postman exported content
- I vote on leaving it, for now, I will remove this fixture soon anyway

Reply:
Unfortunately I have verified that the types for postman collection are actually horrible, and we're going to have a lot of pain because of this. Good luck to us.